### PR TITLE
Slotlist

### PR DESCRIPTION
**Pull request by fish-sushi**

### CODE REVIEW

File: app/javascripts/components/slot/index.jsx [hunk 1]
```diff
@@ -19,7 +21,8 @@ const Slot = ({ match, location, rootState }) =>
     <Switch>
       <Route exact path={`${match.url}/play`} component={PlaySlot} />
       <Route path={`${match.url}/make/:step`} component={MakeGame} />
-      <Route exact path={match.url} component={PlaySlot} />
+      <Route path={`${match.url}/mine`} component={MySlotListContainer} />
```
Comment (parkkoon) [inline app/javascripts/components/slot/index.jsx:24]: 'mine' is good path for expressing mySlotList?

File: app/javascripts/components/slot/layouts/header.jsx [hunk 2]
```diff
@@ -21,10 +21,10 @@ const Header = ({ rootState, location }) =>
         <div className={styles.walletStatus}>
           Your balance : {rootState.get('balance')} ETH
         </div>
-        <button className={styles.accountBtn}>
+        {/* <button className={styles.accountBtn}>
           Your Account
           <Icon className={styles.triangle} icon="TRIANGLE_DOWN" />
-        </button>
+        </button> */}
```
Comment (parkkoon) [inline app/javascripts/components/slot/layouts/header.jsx]: Please delete this comment

**Review by parkkoon — CHANGES_REQUESTED** [review]

File: app/javascripts/components/slot/index.jsx (hunk 1 above)
Comment (fish-sushi) [reply app/javascripts/components/slot/index.jsx:24]: it's temporary decide. we can talk about the paths later

Comment (parkkoon) [reply app/javascripts/components/slot/index.jsx:24]: aha OK!!

**Pull request merged**